File: uri/1 - Basics/1044/Main.java
import java.util.*;

class Main{
    public static void main(String []args){
        Scanner scanner = new Scanner(System.in);

        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int aux;
        if(a < b){
         aux = a;
         a = b;
         b = aux;
        }
        
        if(a%b == 0)
           System.out.printf("Sao Multiplos\n");
        else
           System.out.printf("Nao sao Multiplos\n");
               
    }
}

File: uri/1 - Basics/1044/Main.go
package main

import (
	"fmt"
)

func main() {
	var a, b, aux int

	fmt.Scanf("%d %d", &a, &b)

	if a < b {
		aux = a
		a = b
		b = aux
	}

	if a%b == 0 {
		fmt.Printf("Sao Multiplos\n")
	} else {
		fmt.Printf("Nao sao Multiplos\n")
	}

}
